### DISCUSSION TITLE
Does service accounts need to have permissions to the servers they deploy to?

### DISCUSSION THREAD

**Discussion by Rod-at-DOH** [community/community#125464] [Actions] [answered]
### Select Topic Area

Question

### Body

Our practice, when deploying an ASP.NET Web application from a build server to an internal server, has been to use a Windows service account to deploy to the target server. This has been done longer than I've been at this organization. I assume that the service account had to have the necessary permissions to deploy to IIS on the target server. I'd like to know, is my reasoning correct?

**Comment by indiarasabarreto** [comment]
https://www.tenable.com/audits/items/CIS_Google_Cloud_Platform_v1.2.0_L1.audit:93b5826cec3e7a5d037146c108a7a350

Hope it can help you

**Comment by indiarasabarreto** [comment, marked as answer]
https://stackoverflow.com/questions/26392042/permissions-necessary-to-web-deploy-to-iis-site

**Reply by Rod-at-DOH** [reply]
This is interesting. Everyone where I work is under the impression that the service account used to publish to IIS, must be in the local Administrators group on the target server. So, this is different. Thank you for this information.

**Reply by indiarasabarreto** [reply]
No problem. Glad could help somehow
